components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/pages/cdmf.page.groups/groups.hbs

```diff
@@ -226,8 +226,7 @@
                     <div class="content">
                         <div class="row">
                             <div class="col-lg-5 col-md-6 col-centered">
-                                <h3>Exception at backend. Try Later.</h3>
-
+                                <h3 id="error-msg">Bad Request. Please contact your administrator.</h3>
```

Comment:
AFAIK, it is better to handle all possible errors in client side, regardless that back end is created by our self or any other party. Because if we have necessary error messages and logs in client side, users can identify the issue and can inform the error message to developers. Imagine if someone provide a patch by changing some back end logic or validation, then there could be a chance to get above error. In that case it is necessary to have.